config/connection.php

```diff
@@ -0,0 +1,79 @@
+<?php
+
+/**
+ * @Author : Mfsi_Annapurnaa
+ * @purpose : Db connection operations
+ */
+
+ini_set('display_errors', 1);
+ini_set('display_startup_errors', 1);
+error_reporting(E_ALL);
+
+// Include constants
+include('config/constants.php');
+
+class connection
+{
+
+    static $conn;
+
+    /**
+     * Function for creating database connection
+     *
+     * @access public
+     * @param  void
+     * @return object
+     */
+    function makeConnection()
+    {
+        if (!self::$conn)
+        {
+            // Making connection
+            self::$conn = mysqli_connect(DBHOST, DBUSER, DBPASSWORD, DBNAME);
+
+            // Checking connection
+            if (mysqli_connect_error(self::$conn))
+            {
+                die('Failed to connect to database' . mysqli_connect_error());
```

Comment:
Is it going to show on User interface if there is any error in connection?
May be its the time to start logging errors in error log file? Please check with your mentor on how to start implementing it.